Makefile

```diff
@@ -70,6 +70,10 @@ vendor:
 	$(GO) mod vendor
 	$(GO) mod verify
 
+.PHONY: lint
+lint:
+	find . -name '*.go' -not -path "./vendor/*" | xargs goimports -w
```

Comment:
I thought goimports ignored `./vendor` with `goimports -w ./...`, but that info may be out of date by a few Go versions.

Reply:
That's entirely possible - I've been using gofmt in the majority of Go-based projects, but Nick had said that goimports had been used throughout the OLM projects in the past so this is all a bit new to me. I can track this down further to see if we can avoid having to specify any explicitly excluded directories.

Reply:
Ah, so this is what's documented on [pkg.go.dev](https://pkg.go.dev/golang.org/x/tools/cmd/goimports?utm_source=godoc):

```
To exclude directories in your $GOPATH from being scanned for Go files, goimports respects a configuration file at $GOPATH/src/.goimportsignore which may contain blank lines, comment lines (beginning with '#'), or lines naming a directory relative to the configuration file to ignore when scanning. No globbing or regex patterns are allowed. Use the "-v" verbose flag to verify it's working and see what goimports is doing.
```

I think it's fine to keep using that find command to ignore the vendor directory, and in the future if we need more granularity/control over what files are excluded from goimports, add that `.goimportsignore` file to the root repository.